src/precice/tests/recorder-action-explicit.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+  <log>
+    <sink type="stream" output="stdout"  filter= "%Severity% > debug" format="preCICE:%ColorizedSeverity% %Message%" enabled="true" />	
+  </log>
+
+  <solver-interface dimensions="2" >
+
+    <data:vector name="Forces"  />
+    <data:vector name="Velocities"  />
+
+    <mesh name="MeshOne">
+      <use-data name="Forces" />
+      <use-data name="Velocities" />
+    </mesh>
+
+    <mesh name="MeshTwo">
+      <use-data name="Forces" />
+      <use-data name="Velocities" />
+    </mesh>
+
+    <participant name="SolverOne">
+      <use-mesh name="MeshOne" provide="yes"/>
+      <write-data name="Forces"     mesh="MeshOne" />
+      <read-data  name="Velocities" mesh="MeshOne" />
+      <action:recorder timing="on-time-window-complete-post" mesh="MeshOne">
+      <action:recorder timing="write-mapping-prior" mesh="MeshOne">
+      <action:recorder timing="write-mapping-post" mesh="MeshOne">
+      <action:recorder timing="read-mapping-prior" mesh="MeshOne">
+      <action:recorder timing="read-mapping-post" mesh="MeshOne">
```

Comment:
The more interesting use case will probably be the one where we have this at the participant where also the mapping happens. This one could be a valid check though as well.